app/utils/project.js

```diff
@@ -172,7 +172,11 @@ export default class ProjectUtil {
           f.category === Constants.FilterCategory.FILE_TYPE
         ) {
           // eslint-disable-next-line prettier/prettier
-          const catFilter = f.values.some(v => !v.value && WorkflowUtil.getAssetType(asset) === v.key) ? 2 : 0;
+          const catFilter = f.values.some(
```

Comment:
I'm okay to accept the linter suggestion here, but we should remove the eslint-disable line on 174